main.go

```diff
@@ -59,6 +62,31 @@ func main() {
 	// Initialize the database handler
 	pizzaOven := database.NewPizzaOvenDbHandler(databaseHost, databasePort, databaseUser, databasePwd, databaseDbName)
 
+	// Initializes configuration using a provided yaml file
+	config := &server.Config{NeverEvictRepos: make(map[string]int)}
+	var configParser struct {
+		NeverEvictRepos []string `yaml:"never-evict-repos"`
+	}
+
+	if configPath != "" {
+		configFile, err := os.ReadFile(configPath)
+		if err != nil {
+			sugarLogger.Errorf("No yaml configuration file was provided: %v", err)
+			return
```

Comment:
Let's use `Fatalf` like we do elsewhere in the `main` func.

Also, the error from `os.ReadFile` can be pretty ambiguous. So let's make it abit more general in the message:

```suggestion
sugarLogger.Fatalf("Could not read yaml configuration file: %s", err.Error())
```